pkg/registry/service/registry_test.go

```diff
@@ -236,25 +239,22 @@ func TestEtcdDeleteService(t *testing.T) {
 		t.Errorf("unexpected error: %v", err)
 	}
 
-	if len(fakeClient.DeletedKeys) != 2 {
+	if len(fakeClient.DeletedKeys) != 1 {
```

Comment:
See the first commit of this PR.
Basically, I'm moving "deleting endpoints" from etcd.go into rest.go files. Thus, we longer delete endpoints in the registry (the second key was deleting endpoints). This is now done in rest.go.